nf_core/pipeline-template/docs/usage.md

```diff
@@ -172,104 +172,19 @@ Specify the path to a specific config file (this is a core Nextflow command). Se
-
-1. Check the default version used by the pipeline in the module file for [Pangolin](https://github.com/nf-core/viralrecon/blob/a85d5969f9025409e3618d6c280ef15ce417df65/modules/nf-core/software/pangolin/main.nf#L14-L19)
-2. Find the latest version of the Biocontainer available on [Quay.io](https://quay.io/repository/biocontainers/pangolin?tag=latest&tab=tags)
-3. Create the custom config accordingly:
-
-   - For Docker:
-
-     ```nextflow
-     process {
-         withName: PANGOLIN {
-             container = 'quay.io/biocontainers/pangolin:3.0.5--pyhdfd78af_0'
-         }
-     }
-     ```
+To change the resource requests, please see the [max resources](https://nf-co.re/docs/usage/configuration#max-resources) and [tuning workflow resources](https://nf-co.re/docs/usage/configuration#tuning-workflow-resources) section of the nf-core website.
 
-   - For Singularity:
+### Custom Containers
 
-     ```nextflow
-     process {
-         withName: PANGOLIN {
-             container = 'https://depot.galaxyproject.org/singularity/pangolin:3.0.5--pyhdfd78af_0'
-         }
-     }
-     ```
+In some cases you may wish to change which container or conda environment a step of the pipeline uses for a particular tool. By default nf-core pipelines use containers and software from the [biocontainers](https://biocontainers.pro/) or [bioconda](https://bioconda.github.io/) projects. However in some cases the pipeline specified version maybe out of date.
 
-   - For Conda:
+To use a different container from the default container or conda environment specified in a pipeline, please see the [max resources](https://nf-co.re/docs/usage/configuration#updating-tool-versions) section of the nf-core website.
 
-     ```nextflow
-     process {
-         withName: PANGOLIN {
-             conda = 'bioconda::pangolin=3.0.5'
-         }
-     }
-     ```
+### Custom Tool Arguments
 
-> **NB:** If you wish to periodically update individual tool-specific results (e.g. Pangolin) generated by the pipeline then you must ensure to keep the `work/` directory otherwise the `-resume` ability of the pipeline will be compromised and it will restart from scratch.
+A pipeline might not always support every possible argument or option of a particular tool used in pipeline. Fortunately, nf-core pipelines provide some freedom to users to insert additional parameters that the pipeline does not include by default.
 
-{% if nf_core_configs -%}
```

Comment:
I think we shouldn't remove this line, we want this block of documentation to appear only if configs are enabled. In case I misunderstood and we do want to remove it then we should also remove the closing if [here](https://github.com/nf-core/tools/pull/2201/files#diff-9e117067e5e4219321e75850e6cdce159b9eea7f10bf9864cb6ebac465d57a69R205)

Reply:
Ahh sorry, that was accidently deleted - that's probably why the tests aren't working :face_with_spiral_eyes: